# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/vx32/hotfix.patch
# Copyright (C) 2009 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- src/Makefrag.vanilla	2009-12-16 12:39:17.000000000 +0100
+++ src/Makefrag	2009-12-16 12:39:24.000000000 +0100
@@ -28,7 +28,7 @@
 VX32_CFLAGS := -nostdinc -Ilibvxc/include $(COMMON_CFLAGS)
 VX32_LDFLAGS := -Llibvxc -L$(dir $(shell vx32-gcc -print-libgcc-file-name)) \
 		$(COMMON_LDFLAGS) libvxc/vx32/crt0.o
-VX32_LDLIBS := -lc -lgcc
+VX32_LDLIBS := -lc `$(CC) -print-libgcc-file-name`
 
 # Set compiler options if we should generate x87 FPU code.
 HAVE_X87 = yes
--- src/vxa/Makefrag.vanilla	2009-12-16 12:41:45.000000000 +0100
+++ src/vxa/Makefrag	2009-12-16 12:41:52.000000000 +0100
@@ -4,6 +4,5 @@
 include vxa/bz2/Makefrag
 include vxa/jpeg/Makefrag
 include vxa/jp2/Makefrag
-include vxa/vorbis/Makefrag
 include vxa/flac/Makefrag
 
